guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,268 @@
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of bitcoin-backed ecash is that the tokens are just strings of text, which are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
+
+#### Text Message, Such as Email or SMS:
+Users can send ecash by copying and pasting the token string. This is particularly useful for online transactions or when using text based communication platforms.
+
+#### Bluetooth Transmission:
+Most applications allow users to share a token via Bluetooth. However, more comprehensive Bluetooth sending and receiving functionalities have not yet been fully developed.
+
+#### QR Codes:
+Unlike base chain and Lightning Network QR codes, which only provide directions for where to send Bitcoin, ecash tokens can be embedded within a QR code itself. This means that a user can claim the ecash token by simply scanning the QR code. This method is particularly useful for in person transactions or quick transfers. QR redemption also enables physical bearer assets like paper notes.
+
+
+---
+
+### [Design Best Practices]({{ '/guide/how-it-works/ecash/design-best-practices/' | relative_url }})
+
+Explore guidelines and best practices for optimizing the use of ecash on Bitcoin.
+
+---
+
+## Comparisons and Considerations
+
+### Bitcoin Custody Spectrum
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/custody-spectrum@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/custody-spectrum@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/custody-spectrum@1x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/custody-spectrum@2x.png"
+   alt-text = "Illustration displaying different types of bitcoin custody, showcasing a range from user-held to custodial solutions."
+   caption = "Bitcoin custody options: evaluating decentralization and risk."
+   width = 800
+   height = 456b
+   layout = "full-width"
+%}
+
+{% include tip/open.html color="blue" icon="info" label="Ecash is a custodial solution" %}
+
+Bitcoin-backed ecash is custodial, meaning someone else holds your bitcoin. Be aware that this subjects users to custodial risks.
+
+{% include tip/close.html %}
+
+The spectrum of decentralization and custody for bitcoin ranges from highly decentralized to highly centralized. Holding self-custodial Bitcoin on the base layer is the least risky and most decentralized, rated at 0. Self-custodial Lightning is slightly more centralized at 1, while custodial Lightning moves further towards centralization at 6. ETFs sit at the other extreme, being the most centralized and risky. Ecash improves over custodial Lightning by providing better privacy and security.
+
+### Ecash vs Custodial Lightning
```

Comment:
This is a great suggestion. I moved the info tip up and placed this under it.

`Since ecash is a custodial solution, it’s important to weigh its benefits against the risks of other custodial options. The risk profile of ecash is similar to custodial Lightning, but with some added benefits.`